runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -2765,15 +2765,17 @@ J9::ValuePropagation::transformVTObjectEqNeCompare(TR_OpaqueClassBlock *containi
        *
        * After transformation:
        * n40n   treetop
-       * n39n     arraycmp  <arraycmp>
-       * n100n      aladd
-       * n30n         aloadi  ACMPWideFieldsPrimitive$Holder._wfp QWideFieldsPrimitive;
-       * n28n           ...
-       * n99n         lconst 4
-       * n101n      aladd
-       * n36n         aload  ACMPWideFieldsPrimitive.VALUE1 QWideFieldsPrimitive;
-       * n99n         ==>lconst 4
-       * n98n       iconst 12
+       * n39n     icmpeq
```

Comment:
The example before transformation is `objectInequalityComparison`. Here should be `icmpne`

Reply:
Fixed in commit 5555dc7cdbe1aef429959c6471286298fc4efd4e